process.py

```diff
@@ -210,27 +210,17 @@ def import_to_datastore(directory, provider_code, batch_size):
     goes through all the files in a given directory, parses and commits them
     """
     try:
-        imported = 0
         files_from_lms = dict(get_files(directory))
         if len(files_from_lms) == 0:
             return
         print("importing data from directory: {}".format(directory))
         now = datetime.now()
-        for i, marker in enumerate(import_accidents(provider_code=provider_code, **files_from_lms)):
-            imported = i
-            progress_wheel.show()
-            db.session.add(marker)
-            if i % batch_size == 0 and i > 0:
-                print("\rcommitting ({0} items done)...".format(i))
-                db.session.commit()
-                print("commited.")
-
-        # commit any left sessions, if any were imported
-        if imported > 0:
-            db.session.commit()
-
+        acc = list(import_accidents(provider_code=provider_code, **files_from_lms))
+        print(int((datetime.now() - now).total_seconds()))
```

Comment:
Please make this line more informative - mention that's it's how long (in seconds) it took...